src/data-access/connectDB.ts

```diff
@@ -0,0 +1,5 @@
+import { Sequelize } from "sequelize";
+
+export const sequelize = new Sequelize(
+  "postgres://urcwyrtm:mCkKdNQnNWDgBhDYSgdxkgviQAoZCNd6@tai.db.elephantsql.com:5432/urcwyrtm"
```

Comment:
It is bad practice. It is better to take credentials from ```.env``` file
For example you can use https://www.npmjs.com/package/dotenv

Reply:
dotenv package has been added and used to store the configuration.